linux-omarchy/scripts/backup-omarchy-dotfiles.sh

```
@@ -0,0 +1,112 @@
+
+copy_rel_to_home() {
+  local expanded_path="$1"
+  local rel
+  local display
+
+  if [[ "$expanded_path" == "$HOME" ]]; then
+    rel="."
+    display="~"
+  elif [[ "$expanded_path" == "$HOME/"* ]]; then
+    rel="${expanded_path#"$HOME"/}"
+    display="~/$rel"
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)